llvm/include/llvm/Transforms/Vectorize/LoopVectorizationLegality.h

```diff
@@ -438,6 +466,11 @@ class LoopVectorizationLegality {
   /// Returns true if the loop is vectorizable
   bool canVectorizeMemory();
 
+  /// If LAA cannot determine whether all dependences are safe, we may be able
+  /// to further analyse some unknown dependences and if they match a certain
```

Comment:
There was a recent LAA fix, and the dependences should now be classified as `IndirectUnsafe` I think. Would be good to update the comment

Reply:
done.